frontend/express/public/javascripts/pre-login.js

```diff
@@ -49,6 +49,48 @@ function encodeSomeHtml(html, options) {
     }
 }
 
+
+/**
+* Check password Validation.
+* @param {string} password - password string
+* @return {bool} result - return password is validated or not.
+* @memberof Pre Login
+* 
+/* exported passwordCheck */
+function passwordCheck(password) {
+    if (password.length < 8) {
+        return jQuery.i18n.prop("management-users.password.length", 8);
```

Comment:
I think this localization values would not work here, as that localiztion file is not loaded, we might need separate ones for prelogin?
Or you can recheck how password reset does it.

Reply:
No, it has existed before:
 https://github.com/Countly/countly-server/blob/master/frontend/express/public/localization/pre-login/pre-login.properties#L32-L35